lib/gpu/render_pass.cc

```diff
@@ -96,6 +96,7 @@ void RenderPass::ClearBindings() {
   index_buffer = {};
   index_buffer_type = impeller::IndexType::kNone;
   element_count = 0;
+  scissor.reset();
```

Comment:
The scissor is not a binding. If the user wants to reset it to its original value, they can just set it to the texture size themselves. So no need to reset it here.